src/test/java/io/jenkins/plugins/checks/gitea/GitSCMChecksContextITest.java

```diff
@@ -24,6 +24,11 @@ public class GitSCMChecksContextITest extends IntegrationTestWithJenkinsPerSuite
     private static final String CREDENTIALS_ID = "credentials";
     private static final String URL_NAME = "url";
 
+    @Test
+    public void shouldHaveJenkins() {
+        assertThat(getJenkins()).isNotNull();
```

Comment:
@uhafner Sorry for pining you, but do you have an idea why this simple test fails?

I'm fighting with the integration tests and have no idea how to get further 😕

Reply:
Looks like it's no longer compatible with junit 5?

Reply:
Were those integration tests ever compatible with JUnit 5?

The methods in the Jenkins test harness are most broadly tested and supported with JUnit 4.  JUnit 5 support has been added to some of the methods in the Jenkins test harness, but JUnit 4 is still the preferred version for most Jenkins tests.

Reply:
OK, strange. I don't  explicit reference any JUnit version, so i assume they are comint from parent pom or from `io.jenkins.plugins:plugin-util-api`. 🤔 

I'm now using my own base class with `@Rule JenkinsRule ...` instead of the `IntegrationTestWithJenkinsPerSuite` base class which is working fine.